Cargo.toml

```
@@ -116,6 +117,7 @@ libp2p-identity = { version = "=0.2.9", default-features = false }
 libp2p-ping = { version = "=0.45.0", default-features = false }
 libp2p-swarm = { version = "=0.45.1", default-features = false }
 libp2p-yamux = { version = "=0.46.0", default-features = false }
+pdfium-render = { version = "0.8.25", default-features = true }
```

Same as above basically. `pdfium-render`, however, does [document the features](https://docs.rs/pdfium-render/latest/pdfium_render/#crate-features) and here the default features make sense
```suggestion
pdfium-render = { version = "0.8.25" }
```